light/write.cc

```diff
@@ -915,9 +997,17 @@ void SaveLightmapSurface(const mbsp_t *bsp, mface_t *face, facesup_t *facesup,
             lux = lux_filebase.data() + (id.vanilla_lightofs * 3);
         }
 
+        if (!hdr_filebase.empty()) {
+            hdr = hdr_filebase.data() + (id.vanilla_lightofs * 4);
+        }
+
         // Q2/HL native colored lightmaps
         if (bsp->loadversion->game->has_rgb_lightmap) {
-            lightofs = lit - lit_filebase.data();
```

Comment:
same as above, I think this should stay as `lightofs = lit - lit_filebase.data();`